internal/jujuapi/migrationtarget.go

```diff
@@ -30,13 +30,31 @@ func (api *API) Prechecks(model params.MigrationModelInfo) error
 func init() {
 	facadeInit["MigrationTarget"] = func(r *controllerRoot) []int {
 		preChecks := rpc.Method(r.Prechecks)
+		caCert := rpc.Method(r.CACert)
 
 		r.AddMethod("MigrationTarget", 4, "Prechecks", preChecks)
+		r.AddMethod("MigrationTarget", 4, "CACert", caCert)
 
 		return []int{4}
 	}
 }
 
+// CACert implements the CACert method of the MigrationTarget facade.
+// It is used by the source Juju controller to retrieve the CA cert of
+// the target controller during model migration, if the client did not
+// send a CA cert to the source controller.
+//
+// The above is nonsensical because if the source controller can reach
+// the target controller (and because Juju enforces WSS), it already has
+// everything it needs i.e. it either has the self-signed CA cert or it
+// was able to connect thanks to a public CA.
```

Comment:
...a public CA or central ACME (Like in the case of a usual JIMM deployment).

Reply:
Left this as is